docs/docs/Get-Started/get-started-installation.mdx

```diff
@@ -36,6 +36,12 @@ Langflow Desktop requires macOS 13 or later.
 </TabItem>
 <TabItem value="Windows" label="Windows">
 
+    :::warning
+    Windows users of Langflow Desktop should **not** use the in-app update feature to upgrade to Langflow version 1.6.0.
+    Only Windows Desktop users migrating to Langflow version 1.6.0 are affected.
+    For more information, see [Windows Desktop update issue](/release-notes#windows-desktop-update-issue).
+    :::
```

Comment:
This does not really make sense here because it's the first-time install.
I think I would either remove it or change it to this:

```suggestion
    :::warning
    If you are upgrading Langflow Desktop on Windows, **don't** use the in-app update feature to upgrade to Langflow version 1.6.0.
    For more information, see [Known issues for 1.6.0](/release-notes#windows-desktop-update-issue).
    :::
```